solar-system.rb

```diff
@@ -0,0 +1,80 @@
+#main.rb
+
+require_relative 'planet.rb'
+require_relative 'solar_system.rb'
+
+def main
```

Comment:
For this file to run you would need to comment-out the `require_relative` statements and move the main to the bottom.